test/ts/unit/collateralizer/runners/seize_collateral.ts

```diff
@@ -0,0 +1,243 @@
+                    COLLATERALIZER,
+                    scenario.collateralAmount,
+                );
+
+                // Mock collateral token's allowance for proxy.
+                await mockCollateralToken.mockAllowanceFor.sendTransactionAsync(
+                    COLLATERALIZER,
+                    mockTokenTransferProxy.address,
+                    scenario.collateralAmount,
+                );
+
+                if (scenario.debtAgreementExists &&
+                    scenario.debtAgreementCollateralized &&
+                    scenario.validTermsContract) {
+                    await mockTermsContract.mockCallCollateralize.sendTransactionAsync(
+                        collateralizer.address,
+                        scenario.agreementId,
+                        COLLATERALIZER,
+                    );
+                }
+
+                // 5. Mocking the expected repayment schedule for the agreement id
+                for (const repaymentDate of scenario.expectedRepaymentValueSchedule) {
+                    await mockTermsContract.mockExpectedRepaymentValue.sendTransactionAsync(
+                        scenario.agreementId,
+                        new BigNumber(repaymentDate.timestamp),
+                        repaymentDate.expectedRepaymentValue,
+                    );
+                }
+
+                // 6.  Mocking the current value repaid to date
+                await mockTermsContract.mockDummyValueRepaid.sendTransactionAsync(
+                    scenario.agreementId,
+                    scenario.valueRepaidToDate,
+                );
+
+                // 7.  Mocking the debt term's ending timestamp
+                // await mockTermsContract.mockTermEndTimestamp.sendTransactionAsync(
+                //     scenario.agreementId,
+                //     new BigNumber(scenario.termEndTimestamp),
+                // );
+
+                if (typeof scenario.before !== "undefined") {
+                    await scenario.before(collateralizer, mockTermsContract);
+                }
+
+                ABIDecoder.addABI(collateralizer.abi);
+            });
+
+            after(() => {
+                ABIDecoder.addABI(collateralizer.abi);
+            });
+
+            if (scenario.succeeds) {
+                it("should not throw", async () => {
+                    await expect(
+                        new Promise(async (resolve, reject) => {
+                            try {
+                                txHash = await collateralizer.seizeCollateral.sendTransactionAsync(
+                                    scenario.agreementId,
+                                    { from: scenario.from(BENEFICIARY_1, BENEFICIARY_2) },
+                                );
+
+                                resolve(txHash);
+                            } catch (e) {
+                                reject(e);
+                            }
+                        }),
+                    ).to.eventually.not.be.rejected;
```

Comment:
Let's make this consistent with the other scenario runners -- just making sure a valid txHash is returned